example/ios/AirMapsExplorer/AppDelegate.m

```diff
@@ -11,12 +11,16 @@
 
 #import "RCTRootView.h"
 
+#import <GoogleMaps/GoogleMaps.h>
+
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
   NSURL *jsCodeLocation;
 
+  [GMSServices provideAPIKey:@"AIzaSyAeHIC4IG7XKT2Ls5Ti_YZV-6DHQk6dVHE"];
```

Comment:
Interesting.  Would the key you generated work with Android? It would be nice to just have one key for the whole example app.